config/config.go

```diff
@@ -33,6 +33,11 @@ type Repo struct {
 	EnablePushUpdates *bool          `json:"enable-push-updates"`
 }
 
+type RepoDefaults struct {
```

Comment:
The name of the struct should be singular. So, we can have `RepoDefaultSetting` or `RepoDefault` here.